### PR TITLE
Extensions for WikiCanada

### PR DESCRIPTION
**Pull request by Amanda1234567**

### CODE REVIEW

File: LocalSettings.php [hunk 1]
```diff
@@ -1053,6 +1053,7 @@
 	'wmgUseDismissableSiteNotice' => array(
 		'default' => true,
 		'extloadwiki' => true,
+		'wikicanadawiki' => false;
```
Comment (Reception123) [inline LocalSettings.php]: use a , not a ;

**Comment by Amanda1234567** [comment]
Fixed

**Comment by Reception123** [comment]
@Amanda1234567 Parsoid is also not enabled for flow.

**Comment by Amanda1234567** [comment]
I didn't enable flow

**Comment by Reception123** [comment]
Sorry, nevermind. Waiting for Travis CI to confirm

**Comment by Amanda1234567** [comment]
@Reception123 Travis is taking forever today - John merged the other one before it finished and it seems fine

**Comment by Amanda1234567** [comment]
@Reception123 @JohnFLewis Travis finally finished - please merge

**Pull request merged**